apps/hash-frontend/src/blocks/page/create-suggester/mention-suggester.tsx

```diff
@@ -1,188 +1,632 @@
+import { useQuery } from "@apollo/client";
+import { VersionedUrl } from "@blockprotocol/type-system";
+import { LoadingSpinner } from "@hashintel/design-system";
+import {
+  currentTimeInstantTemporalAxes,
+  generateVersionedUrlMatchingFilter,
+  zeroedGraphResolveDepths,
+} from "@local/hash-isomorphic-utils/graph-queries";
 import { types } from "@local/hash-isomorphic-utils/ontology-types";
-import { EntityId, OwnedById } from "@local/hash-subgraph";
-import { getEntityTypeById, getRoots } from "@local/hash-subgraph/stdlib";
-import { Box } from "@mui/material";
-import { FunctionComponent, useContext, useMemo } from "react";
-
-import { useAccountPages } from "../../../components/hooks/use-account-pages";
-import { useQueryEntities } from "../../../components/hooks/use-query-entities";
-import { useUsers } from "../../../components/hooks/use-users";
-import { PageIcon } from "../../../components/page-icon";
+import {
+  Entity,
+  EntityId,
+  EntityRootType,
+  EntityTypeWithMetadata,
+  OwnedById,
+  Subgraph,
+} from "@local/hash-subgraph";
+import {
+  getEntityTypeById,
+  getOutgoingLinkAndTargetEntities,
+  getRoots,
+} from "@local/hash-subgraph/stdlib";
+import { List, ListItem, ListItemIcon, ListItemText } from "@mui/material";
+import {
+  Fragment,
+  FunctionComponent,
+  useCallback,
+  useEffect,
+  useMemo,
+  useRef,
+  useState,
+} from "react";
+import { useKey } from "rooks";
+
+import { useScrollLock } from "../../../components/grid/utils/override-custom-renderers/use-scroll-lock";
```

Comment:
Can you move this into a shared folder please?

Reply:
Yes, moved in https://github.com/hashintel/hash/pull/3244/commits/2be6c078db5a5f6395a2d865b8e6ac78d05de616